src/common/copilot/assets/scripts/copilot.js

```diff
@@ -575,11 +577,11 @@
   function handleFeedbackClick(event) {
     const target = event.target;
 
-    if (target.classList.contains(THUMBS_UP)) {
+    if (target.classList.contains("thumbsup")) {
```

Comment:
if we are comparing with direct tring - due we need the declaration above?

Reply:
Yes, that declaration is for feedback values (camelCase) and this string comparison is for class name.